spec/spec_helper.rb

```diff
@@ -1,25 +1,35 @@
 # Require this file for unit tests
-ENV['LOTUS_ENV'] ||= 'test'
+ENV["LOTUS_ENV"] ||= "test"
 
-if ENV['COVERALLS']
-  require 'coveralls'
+if ENV["COVERALLS"]
+  require "coveralls"
   Coveralls.wear!
 end
 
-if ENV['COVERAGE']
-  require 'simplecov'
+if ENV["SCRUTINIZER"]
+  require "scrutinizer/ocular"
+  Scrutinizer::Ocular.watch!
+end
+
+if ENV["COVERAGE"]
+  require "simplecov"
 
   SimpleCov.start do
     add_filter "/spec/"
+    add_group "Lib", "lib"
     add_group "Quizzy", "lib/quizzy"
     add_group "Apps", "apps"
     add_group "Web", "apps/web"
-    add_group "Controllers", "apps/web/controllers"
-    add_group "Views", "apps/web/views"
+    add_group "Admin", "apps/admin"
+    add_group "Entities", "lib/quizzy/entities"
+    add_group "Repositories", "lib/quizzy/repositories"
   end
 end
 
-require_relative '../config/environment'
-require 'minitest/autorun'
+require_relative "../config/environment"
+require "minitest/autorun"
+require 'minitest/unit'
+require 'mocha/mini_test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.